### ISSUE TITLE
Sidebar should be hidden by default

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/sul-embed#1734]
Related to #1726 and #1731 .

By default (upon loading the page), the Sidebar should be hidden from the accessibility tree: `visibility: hidden;`. Currently, it is only hidden when the user toggles the button to collapse it.

**Comment by jcoyne** [comment]
Ooops, I had made this change, but it didn't get pushed up to git before the PR was merged. Thank you for catching this.

**Comment by mjgiarlo** [comment]
@jcoyne If you have this commit hanging around, I'll move this to In Progress.
